ovos_PHAL_plugin_homeassistant/__init__.py

```diff
@@ -110,9 +110,8 @@ def __init__(self, bus=None, config=None):
         self.bus.on("configuration.patch", self.init_configuration)
 
         # LISTEN FOR OAUTH RESPONSE
-        self.bus.on("oauth.app.host.info.response", self.handle_oauth_host_info)
-        self.bus.on("oauth.generate.qr.response", self.handle_qr_oauth_response)
```

Comment:
Did you mean to remove these? Is that because the OAuth plugin handles them?

Reply:
Yeah, these were refactored so `wait_for_response` will handle these during the flow in one thread. Couple reasons I opted for this:
- prevents handling events if another service is performing oauth
- allows handling one oauth flow in one thread so raised exceptions get to the original method
- allows raising timeout errors/knowing when an expected response didn't come